src/digraph.rs

```diff
@@ -1526,7 +1531,7 @@ impl PyDiGraph {
 
     /// Add a new node to the graph.
     ///
-    /// :param obj: The python object to attach to the node
+    /// :param S obj: The python object to attach to the node
```

Comment:
```suggestion
    /// :param S obj: The Python object to attach to the node
```

Reply:
I believe I left lowercase pythons in `graph.rs` too.